contentpal/src/main/java/org/dmfs/android/contentpal/rowdata/SimpleRowData.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.android.contentpal.rowdata;
+
+import android.content.ContentProviderOperation;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.dmfs.android.contentpal.RowData;
+import org.dmfs.android.contentpal.TransactionContext;
+
+
+/**
+ * {@link RowData}s that simply adds the provided value with the given key.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class SimpleRowData<T> implements RowData<T>
```

Comment:
How about the name `RawRowData` to indicate that this is sort of a low-level thing and should be used with care. The major concern would be that, since it takes and object, someone uses it to pass non-primitive values.

Reply:
Right, the danger occured to me, too. So `RawRowData` indeed sounds better because of that. I've renamed it and also added warning in the docs, listing the supported types.